3rdParty/velocypack/src/Validator.cpp

```diff
@@ -121,14 +129,18 @@ bool Validator::validate(uint8_t const* ptr, std::size_t length, bool isSubPart)
     }
 
     case ValueType::Array: {
-      ++_level;
+      if (++_level >= options->nestingLimit) {
+        throw Exception(Exception::TooDeepNesting);
```

Comment:
Is there any chance we can improve this error messages?
When debugging such an issue any additional context will help.
To reach this error the document to test typically is more complex, so e.g. char position or attributePath where the validator fails will be very helpful.